BasiliskII/src/Unix/video_x.cpp

```diff
@@ -812,6 +838,22 @@ driver_window::driver_window(X11_monitor_desc &m)
 	wattr.border_pixel = 0;
 	wattr.colormap = (mode.depth == VDEPTH_1BIT ? DefaultColormap(x_display, screen) : cmap[0]);
 
+#ifdef ENABLE_XRENDER
+	{
+		double xres = (((double) DisplayWidth(x_display, 0)) * 25.4) /
+							((double) DisplayWidthMM(x_display, 0));
+		double yres = (((double) DisplayHeight(x_display, 0)) * 25.4) /
+							((double) DisplayHeightMM(x_display, 0));
+		int dpi = (xres + yres) / 2;
+
+		if (dpi > 100) {
```

Comment:
BTW, I'm working on another PR to enhance the scale. 

Instead of making it full screen (as I suggest above), you can scale up or down the guest OS window at will by adjusting the windows size. At any time, the guest OS is in the pre-defined virtual resolution. But it can scale up and down easily.

It is quite difficult to add a menu to SDL2. So I will use this way to restore 1:1 ratio (i.e. no scale up or scale down): first minimize the window and then pops up the window, again.